modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,110 @@
+      template:
+        metadata:
+          labels:
+            machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+            machine.openshift.io/cluster-api-machine-role: <role> <2>
+            machine.openshift.io/cluster-api-machine-type: <role> <2>
+            machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+        spec:
+          metadata:
+            labels:
+              node-role.kubernetes.io/<role>: "" <2>
+          providerSpec:
+            value:
+              apiVersion: ovirtproviderconfig.machine.openshift.io/v1beta1
+              cluster_id: <ovirt-cluster-id> <6>
+              template_name: <ovirt-template-name> <7>
+              instance_type_id: <instance-type-id> <8>
+              cpu: <9>
+                cores: <number-of-cores> <10>
+                sockets: <number-of-sockets> <11>
+                threads: <number-of-threads> <12>
+              memory_mb: <memory-size> <13>
+              os_disk: <14>
+                size_gb: <disk-size> <15>
+              network_interfaces: <16>
+                vnic_profile_id:  <vnic_profile_id> <17>
+              credentialsSecret:
+                name: ovirt-credentials <18>
+              kind: OvirtMachineProviderSpec
+              type: <workload-type> <19>
+              userDataSecret:
+                name: worker-user-data
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI (oc) installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+<2> Specify the node label to add.
+
+<3> Specify the infrastructure ID and node label.
+
+<4> Specify the number of machines to create.
+
+<5> Selector for the machines.
+
+<6> Specify a UUID for the oVirt cluster to which this VM instance belongs.
+
+<7> Specify the oVirt VM template to use to create the machine.
+
+<8> (Optional) Specify the VM instance type. If you include this parameter, you do not need to specify the hardware parameters of the VM including CPU and memory because this parameter overrides all hardware parameters.
+
+<9> (Optional) The CPU field contains configuration for the VM cpu, made of (Sockets * Cores * Threads).
+
+<10> (Optional) Specify the number of sockets for a VM.
+
+<11> (Optional) Specify the number of cores per socket.
+
+<12> (Optional) Specify the number of threads per core.
+
+<13> (Optional) Specify the size of a VM's memory in MiBs.
+
+<14> (Optional) Root disk of the node.
+
+<15> (Optional) Specify the size of the bootable disk in GiB.
+
+<16> (Optional) List of the network interfaces of the VM. If you include this parameter, OCP discards all network interfaces from the template and creates new ones.
+
+<17> (Optional) Specify the vNic profile ID.
```

Comment:
Is "vNic" the correct capitalization? I see vNIC elsewhere in the docs.

Reply:
Fixed.